docs/templates/plugin.rst.j2

```diff
@@ -117,9 +117,9 @@ Parameters
                     <div class="cell-border">
                         {# Turn boolean values in 'yes' and 'no' values #}
                         {% if value.default is defined %}
```

Comment:
This is no longer needed, because comparing to true or false using the sameas test will take care of undefined values as well. It won't cause an exception.

Reply:
The gory details are available from: https://github.com/ansible/ansible/pull/37517/files#diff-6f1bfe955228c54cd94db92b54b402d8

Reply:
Changed that.